src/components/Category.js

```diff
@@ -85,19 +109,29 @@ const CategoryComponent = ({ category, stateContext }) => {
     ? COLUMNS
     : COLUMNS.filter((x) => x.key !== "State")
 
+  const onSearchChange = (e) => {
+    const currValue = e.target.value
+    setSearchedValue(currValue)
+  }
```

Comment:
Not a big deal, but we probably don't need to declare a const 
```
const onSearchChange = (e) => setSearch(e.target.value)
```

Reply:
Updated this